clientd3d/config.c

```diff
@@ -102,6 +102,11 @@ static char INIDebug[]        = "Debug";
 static char INISecurity[]     = "Security";
 static char INITechnical[]    = "Technical";
 
+static char config_ini_file[] = "./config.ini";
+static char config_section[] = "config";
```

Comment:
Add a comment specifying what this section is for (see the other section definitions).